src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke.command;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Command to exit the system and say bye.
+ */
+public class ByeCommand extends Command {
+	/**
+	 * Executes the bye command which exits system.
```

Comment:
Perhaps there should be an empty line between description and parameter section? (applies to other Javadoc comments too)